packages/react-ui-components/package.json

```diff
@@ -9,7 +9,7 @@
   "main": "./src/index",
   "scripts": {
     "build-standalone-esm": "rm -Rf .postcssrc lib-esm && tsc --build tsconfig.esm.json && tsc --build tsconfig.esmtypes.json && rsync --prune-empty-dirs  -r --include='*/' --include='*.css' --include='*.svg' --exclude='*' src/ lib-esm -v && node scripts/generate-standalone-postcssrc.js",
```

Comment:
hmm we might need this now due to  https://github.com/neos/neos-ui/pull/3200#issuecomment-1281229469

--include='*.scss'